### PR TITLE
feat(i18n): add `manage lists` spanish translation

### PR DESCRIPTION
**Pull request by userquin**
This PR also fix errata in `list.edit` entry.

/cc  @patak-dev @mrcego

### CODE REVIEW

File: locales/es.json [hunk 1]
```diff
@@ -217,11 +217,12 @@
     "create": "Crear",
     "delete": "Eliminar esta lista",
     "delete_error": "Se produjo un error eliminando la lista",
-    "edit": "Ediar esta lista",
+    "edit": "Editar esta lista",
     "edit_error": "Se produjo un error modificando la lista",
     "error": "Se produjo un error creando la lista",
     "error_prefix": "Error: ",
     "list_title_placeholder": "Título de la lista",
+    "manage": "Administrar listas",
```
Comment (mrcego) [inline locales/es.json:225]: En plural?

Comment (userquin) [reply locales/es.json:225]: Si, la original es `Manage lists`

Comment (userquin) [reply locales/es.json:225]: https://github.com/elk-zone/elk/blob/main/locales/en.json#L225

**Comment by mrcego** [comment]
LGTM 💚

**Review by patak-dev — APPROVED** [review]

**Pull request merged**